@xen-orchestra/web-core/lib/components/state-hero/VtsLoadingHero.vue

```diff
@@ -1,13 +1,51 @@
 <template>
-  <VtsStateHero :type busy>
-    {{ $t('loading-in-progress') }}
-  </VtsStateHero>
+  <div class="vts-loading-hero">
+    <VtsStateHero :type busy>
+      {{ $t('loading-in-progress') }}
+    </VtsStateHero>
+    <div class="content">
+      <div v-if="slots.title" class="title" :class="typoClass">
+        <slot name="title" />
+      </div>
+      <div v-if="slots.text" class="text typo-body-bold">
+        <slot name="text" />
+      </div>
+    </div>
+  </div>
 </template>
 
 <script lang="ts" setup>
 import VtsStateHero, { type StateHeroType } from '@core/components/state-hero/VtsStateHero.vue'
+import { computed } from 'vue'
 
-defineProps<{
+const props = defineProps<{
   type: StateHeroType
 }>()
+
+const slots = defineSlots<{
+  title(): string
+  text(): string
+}>()
+
+const typoClass = computed(() => (props.type === 'page' ? 'typo-h1' : 'typo-h2'))
```

Comment:
Not really in the guidelines for now, but to keep consistency, we tend to name this kind of _computed_ as `className`:
```suggestion
const className = computed(() => (props.type === 'page' ? 'typo-h1' : 'typo-h2'))
```